src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -116,7 +116,7 @@ const SidebarLinks = (props) => {
                                 text: participantDetails ? participantDetails.displayName : report.reportName,
                                 alternateText: participantDetails ? participantDetails.login : '',
                                 type: participantDetails ? 'user' : 'report',
-                                icon: participantDetails ? participantDetails.avatarURL : '',
+                                icon: report.icon,
```

Comment:
NAB, could we maybe leave a comment here to explain why the icon is on the report? I think this is being worked on by some contractors and it's not super obvious. Something like 

>The icon for the row is set when the fetching personal details via `PersonalDetails.getFromReportParticipants()`